news/2022-11-12-osu-singapore-aef-online-2022.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "osu! Singapore: AEF Online 2022"
+date: 2022-11-12 04:00:00 +0000
```

Comment:
that would be saturday, i.e. featured artist day (@Ephemeralis for scheduling, apparently registrations are supposed to open that day)